src/components/Content/Content.jsx

```diff
@@ -32,49 +37,58 @@ function Content() {
       movieServices.getById(url)
         .then((data) => {
           if (data) {
-            setCurrentMovie(data);
+            // eslint-disable-next-line react/destructuring-assignment
+            props.setCurrentMovie(data);
           }
         });
     }
   }, [currentMovieId]);
 
   const getCurrentMovieId = (e) => {
-    setCurrentMovieId(() => e.target.id);
-  };
-
-  const filterByLikes = () => {
-    const newList = [...moviesList].sort((a, b) => b.likes - a.likes);
-    setSearchList(newList);
+    setCurrentMovieId(e.target.id);
   };
 
-  const filterByRating = () => {
-    const newList = [...moviesList].sort((a, b) => b.rating - a.rating);
-    setSearchList(newList);
-  };
-
-  const setLikesById = (id, likes) => {
-    const movie = moviesList.find((item) => item.id === id);
-    const idx = moviesList.findIndex((item) => item.id === id);
-    movie.likes = likes;
-    moviesList.splice(idx, 1, movie);
-    setMoviesList(moviesList);
+  const sortingByLikes = (e) => {
+    if (e.target.value === 'ASC') {
+      const newList = [...searchList].sort((a, b) => b.likes - a.likes);
+      setSearchList(newList);
+      setSortingType(e.target.value);
+    }
+    if (e.target.value === 'DESC') {
+      const newList = [...searchList].sort((a, b) => a.likes - b.likes);
+      setSearchList(newList);
+      setSortingType(e.target.value);
+    }
+    if (!e.target.value) {
+      setSearchList(movies);
+      setSortingType('');
+    }
   };
 
-  const setRatingById = (id, rating) => {
-    const movie = moviesList.find((item) => item.id === id);
-    const idx = moviesList.findIndex((item) => item.id === id);
-    movie.rating = rating;
-    moviesList.splice(idx, 1, movie);
-    setMoviesList(moviesList);
+  const sortingByRating = (e) => {
+    if (e.target.value === 'ASC') {
```

Comment:
For me, the best idea is to make sort function as separate function from utils with passing filter param and the direction of sorting. sortMovies('rating', 'ASC')